elementary/monitor/api/tests/tests.py

```diff
@@ -41,16 +41,6 @@ def __init__(self, dbt_runner: DbtRunner):
         super().__init__(dbt_runner)
         self.invocations_api = InvocationsAPI(dbt_runner)
 
-    @staticmethod
-    def get_test_sub_type_unique_id(
```

Comment:
Moved into current_tests_run_results_query macro to simplify it and having one place that generates the `test_sub_type_unique_id`